src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1098,13 +1105,13 @@ public String toString() {
 
   // TieBreakMode determines how ties will be handled
   enum TieBreakMode {
-    RANDOM("random"),
-    INTERACTIVE("interactive"),
-    PREVIOUS_ROUND_COUNTS_THEN_RANDOM("previousRoundCountsThenRandom"),
-    PREVIOUS_ROUND_COUNTS_THEN_INTERACTIVE("previousRoundCountsThenInteractive"),
-    USE_PERMUTATION_IN_CONFIG("usePermutationInConfig"),
-    GENERATE_PERMUTATION("generatePermutation"),
-    MODE_UNKNOWN("modeUnknown");
+    RANDOM("Random"),
+    INTERACTIVE("Stop counting and ask"),
+    PREVIOUS_ROUND_COUNTS_THEN_RANDOM("Previous round counts (then random)"),
```

Comment:
```suggestion
    PREVIOUS_ROUND_COUNTS_THEN_RANDOM("Previous round tallies then random"),
```

Reply:
Per our conversation, will leave all labels as the UX person recommended for now.